keps/sig-node/4210-max-image-gc-age/README.md

```diff
@@ -208,12 +208,13 @@ Additional tests will be added to this file to cover the garbage collection e2e.
 
 - Configuration field added to the Kubelet (disabled by default)
 - Feature supported by Kubelet Image Manager
-- Unit tests and e2e tests added
+- Unit tests
 - Add a metric `kubelet_image_garbage_collected_total` which tracks the number of images the kubelet is GC'ing through any mechanism.
 
 #### Beta
```

Comment:
does metric needs to be promoted to stability level BETA? Is this stability level supported? If so - please add as a graduation criteria

Reply:
all of kubelet's metrics are in ALPHA

Reply:
all kubelet metrics are ALPHA